src/core/positioning/bluetoothdevicemodel.h

```diff
@@ -61,8 +61,10 @@ class BluetoothDeviceModel : public QAbstractListModel
 
     QHash<int, QByteArray> roleNames() const override;
 
+    Q_INVOKABLE int addDevice( const QString &name, const QString &address );
```

Comment:
Docstring (adds if not exists, returns index of ...)

Reply:
@m-kuhn , good point. Big thanks for the swift review!